release/gcb/helm_charts.sh

```diff
@@ -29,6 +29,10 @@ cp -R "./istio-${CB_VERSION}/install" "$WORK_DIR/istio/install"
 
 pushd "$WORK_DIR"
     git clone -b "${CB_BRANCH}" https://github.com/istio/cni.git
+    if [ "${CB_PIPELINE_TYPE}" = "monthly" ]; then
```

Comment:
why only monthly?

Reply:
Because for cni we only have master-latest-daily tag in gcr.io/istio-release. 
Which is different from other releases, for other releases there are daily-build tags.

Reply:
I downloaded the daily build and see a tag:
`docker pull gcr.io/istio-release/install-cni:release-1.2-20190613-09-15`

Reply:
I was looking into cni not instal-cni, not confusing at all :)

Reply:
Updated the PR

Reply:
I think that is the only image (just from looking at the release at least), not sure where `cni` is from? Maybe it is just an old one

Reply:
Yup, definitely old one.